### PR TITLE
feat: update quick-start guide to use the new CLI

### PR DESCRIPTION
**Pull request by brent-hoover**
Signed-off-by: Brent Hoover <brent@thebuddhalodge.com>

### CODE REVIEW

File: guides/quick-start.md [hunk 1]
```diff
@@ -6,43 +6,57 @@ In this guide, we’ll set up a full local instance of the Open Commerce platfor
...
-```
-
-> **Note**: `make` starts containers in production mode, so you won't be able to make changes to the code and see them. For running in development mode, use `make init-dev`. See list of all commands [here](https://github.com/reactioncommerce/reaction-development-platform#project-commands)
+In addition you need to have your system setup for [SSH authentication with GitHub](`https://docs.github.com/en/authentication/connecting-to-github-with-ssh`)
 
-> **Note**: On Windows, you may need to replace `make` with `sudo make`. Then run `chown -R` on the `reaction-development-platform` directory to set your user as the owner of all the files it contains.
+## Install the CLI
 
-Behind the scenes, the `make` process clones all of the relevant Open Commerce software repositories, sets up each environment, and pulls, builds, and starts each Docker container in production mode.
+`npm install -g @reactioncommerce/reaction-cli`
```
Comment (tedraykov) [inline guides/quick-start.md]: This installs the deprecated cli. It should be replaced with `npm uninstall -g reaction-cli`.

File: guides/quick-start.md [hunk 2]
```diff
@@ -6,43 +6,57 @@ In this guide, we’ll set up a full local instance of the Open Commerce platfor
...
+## Install the API Server
 
-- Open Commerce API (port 3000), including the [core plugins](/developer/open-commerce/docs/plugins-environment-variables/#plugins). This service also contains the GraphQL playground at `localhost:3000/graphql`.
+- Create a directory for your entire project using `mkdir myproject` and then `cd` into it
+- Create an API server by running `reaction create-project api <myapiserver>`, _you can substitute any directory 
+  name for `<myapiserver>`_
+- Change directory into your newly created server directory and run `npm install`
+- Once this is complete run `reaction develop`. This will start the Open Commerce server in development mode.
+- Congrats you have installed the Mailchimp Open Commerce API server. You can now proceed to install the Storefront 
+  and Admin applications. You can view the graphQL playground locally `http://localhost:3000/graphql`
```
Comment (tedraykov) [inline guides/quick-start.md]:
```suggestion
  and Admin applications. You can view the GraphQL playground locally `http://localhost:3000/graphql`
```

**Review by tedraykov — CHANGES_REQUESTED** [review]

File: guides/quick-start.md (hunk 1 above)
Comment (brent-hoover) [reply guides/quick-start.md]: This does not install the deprecated CLI, have you tested it lately?

Comment (brent-hoover) [reply guides/quick-start.md]: It may have installed the old CLI in the past, but since the new one has been published this command works correctly

Comment (tedraykov) [reply guides/quick-start.md]: Screenshot 2022-08-22 at 12 25 44
https://www.npmjs.com/package/@reactioncommerce/reaction-cli

Comment (brent-hoover) [reply guides/quick-start.md]:
```
 »» npm install -g reaction-cli
/home/brent/.nvm/versions/node/v14.18.1/bin/reaction -> /home/brent/.nvm/versions/node/v14.18.1/lib/node_modules/reaction-cli/index.js
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@^2.3.2 (node_modules/reaction-cli/node_modules/jest-haste-map/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.3.2: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

+ reaction-cli@1.2.1
added 107 packages from 20 contributors and updated 461 packages in 37.566s
[brent@ferret: 17:21:32 ~/Projects/merchstack/docs  (getting-started-cli) - [2015]
 »» reaction help

Comment (brent-hoover) [reply guides/quick-start.md]: Also why would I issue the uninstall command to install the CLI?

Comment (brent-hoover) [reply guides/quick-start.md]: Ah I see, need to drop the org for some reason

**Comment by brent-hoover** [comment]
This was (mostly) updated in ContentStack. The plugin list updates were not

**Pull request merged**